src/PrestaShopBundle/Command/ListCommandsAndQueriesCommand.php

```diff
@@ -62,27 +86,111 @@ public function configure()
         $this
             ->setName('prestashop:list:commands-and-queries')
             ->setDescription('Lists available CQRS commands and queries')
+            ->addOption(
+                'domain',
+                'd',
+                InputOption::VALUE_IS_ARRAY | InputOption::VALUE_REQUIRED,
+                'Filter available CQRS by domain.'
+            )
+            ->addOption(
+                'format',
+                'f',
+                InputOption::VALUE_REQUIRED,
+                'Outputs either a regular or simplified format.',
+                'regular'
+            )
         ;
     }
 
     /**
      * {@inheritdoc}
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $this->apiResourcesList = $this->getResourceList();
+        $this->handleOptions($input);
+
         $outputStyle = new OutputFormatterStyle('blue', null);
         $output->getFormatter()->setStyle('blue', $outputStyle);
 
         foreach ($this->commandAndQueries as $key => $commandName) {
             $commandDefinition = $this->commandDefinitionParser->parseDefinition($commandName);
+            $isCQRSImplemented = $this->isCQRSImplemented($commandDefinition);
 
-            $output->writeln(++$key . '.');
-            $output->writeln('<blue>Class: </blue><info>' . $commandDefinition->getClassName() . '</info>');
-            $output->writeln('<blue>Type: </blue><info>' . $commandDefinition->getCommandType() . '</info>');
-            $output->writeln('<comment>' . $commandDefinition->getDescription() . '</comment>');
-            $output->writeln('');
+            if ($this->isFormatSimple) {
+                $output->writeln('<info>' . $commandDefinition->getClassName() . ($isCQRSImplemented !== '' ? ' OK' : ' NOT OK') . '</info>');
```

Comment:
`OK` or `NOT OK` seems not cristal clear to me.
Can we find another terms?

Reply:
This is the way it was asked in the original issue.